lib/files-callBack.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-undef */
+'use strict';
+// eslint-disable-next-line no-undef
+const fs = require('fs');
+// eslint-disable-next-line no-undef
+
+// eslint-disable-next-line no-unused-vars
+const util = require('util');
```

Comment:
You're not using `util` in this file, so you don't need to import it.

Reply:
it has been removed